app/src/main/java/ar/edu/unlam/mobile/scaffold/ui/components/Grafic.kt

```diff
@@ -0,0 +1,140 @@
+package ar.edu.unlam.mobile.scaffold.ui.components
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.scale
+import androidx.compose.ui.graphics.nativeCanvas
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.sp
+import ar.edu.unlam.mobile.scaffold.domain.models.PieChartInput
+import androidx.compose.material3.Text as Text
+@Composable
+fun pieChart(
+    data: List<PieChartInput>,
+    radiousOuter: Float = 500f,
+    innerRadious: Float = 250f,
+    transparentwidth: Float = 70f,
+) {
+    val totalSum =
```

Comment:
```suggestion
    val totalSum = data.sumOf { it.value }
```
Se puede reducir lineas